content/en/blog/_posts/2022-02-25-K8s Triage bot Helper CI Job_ ‘make update’.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: blog
+title: "K8s Triage bot Helper CI Job: ‘*make update*’"
+date: 2022-02-25
+slug: k8s-triage-bot-helper-ci-job
+---
+
+# K8s Triage bot Helper CI Job: ‘*make update*’
+
+## Introduction 
+
+If you are contributing to Kubernetes.io and developing on a Windows PC, it is conceivable that you will encounter certain issues that will cause your patches to master to somehow be halted. This blog is a workaround for a similar issue I encountered when attempting to have my modifications approved and merged into the master repository.
```

Comment:
```suggestion
If you are contributing to the Kubernetes project and are developing on a Windows PC, it is conceivable that you will encounter certain issues that will cause your patches to master to somehow be halted. This blog is a workaround for a similar issue I encountered when attempting to have my modifications approved and merged into the master repository.
```

Reply:
I think it should be the Kubernetes project because kubernetes.io is the website and might give the audience a wrong impression of the content we're speaking about.

Reply:
seems better now, thanks @divya-mohan0209